changelogs/unreleased/8485-drop-deprecated-logging-api-methods.yml

```
@@ -0,0 +1,10 @@
+description: >
+  Drop deprecated methods: `set_log_level`, `set_log_formatter`, `set_logfile_location` and `get_handler`
+  from the stable api of the logging.
```

Not sure about the wording and relevance for end-user

---

I would also mentions the class we removed it from:

```suggestion
  Dropped deprecated methods `set_log_level`, `set_log_formatter`, `set_logfile_location` and `get_handler`
  from `inmanta.logging.InmantaLoggerConfig`. Use the `apply_options` method or a log config file instead. 
```